### DISCUSSION TITLE
13D974FA-19C2-9460-C628-A292E83A8BA8

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#57] [General]
# Jupiter Icy Moons Explorer

The Jupiter Icy Moons Explorer (JUICE) spacecraft is a future European planetary flagship mission that is currently in development and scheduled for launch on an Ariane 5 rocket by April 2023. Born out of the former Jupiter Ganymede Orbiter concept, it will study three of the Galilean satellites from 2031 to 2034 (Europa, Ganymede, and Callisto) before entering orbit around Ganymede by 2035. It will complement NASA's Europa Clipper, which is also scheduled to study the Jovian system at the same time period, thereby fulfilling the objectives first proposed on the cancelled NASA/ESA mission in 2009 called the Europa-Jupiter System Mission

https://celestia.mobi/addon?item=13D974FA-19C2-9460-C628-A292E83A8BA8&lang=zh_CN

**Comment by 13940951572** [comment]
你好这是实时的吗？

**Reply by DaveBowman2001** [reply]
Currently, the trajectory this addon uses is the prelaunch flight profile as planned back in 2022. The newer postlaunch actual trajectory data (available on [JPL HORIZONS](https://ssd.jpl.nasa.gov/horizons/)) omitted the Jupiter/Ganymede orbiting phases on 2031-2034, so it might take a while before a proper update is released

**Reply by 13940951572** [reply]
JPL地平线在哪能看到！设置？

**Reply by ASTIDA114514** [reply]
你好兄弟，JPL地平线网址为https://ssd.jpl.nasa.gov/horizons/manual.html，但国内好像无法进入

**Comment by bird0cosmic** [comment]
Cool

**Comment by Breackman** [comment]
agree

**Comment by exocritic5457** [comment]
rip lunchables
